debian/control

```diff
@@ -0,0 +1,30 @@
+Source: gtklock
+Section: unknown
+Priority: optional
+Maintainer: Soumya Ranjan Patnaik <soumyaranjan1812@gmail.com>
+Rules-Requires-Root: no
+Build-Depends:
+ debhelper-compat (>= 11),
```

Comment:
I changed this from `=13` to `>= 11` due to https://github.com/regolith-linux/voulage/actions/runs/3975629370/jobs/6815610614